.github/workflows/workflow.yml

```diff
@@ -82,13 +108,14 @@ jobs:
         python-version: [3.9]
 
     timeout-minutes: 30
+
     steps:
-    - uses: actions/checkout@master
-    - uses: actions/setup-python@master
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
-      run: pip install tox==3.24.1
-    - name: Integration tests
+      run: pip install tox==3.25.1
+    - name: Integration Tests
       run: |
           tox -e test-integration
```

Comment:
Similar comment as unit-test above re. removing unwanted deps

Reply:
done in dependency PR